cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -496,6 +557,37 @@ runGenesisCreateTestNetDataCmd
 desiredKeyOutputFormat :: KeyOutputFormat
 desiredKeyOutputFormat = KeyOutputFormatTextEnvelope
 
+-- | We need to pass these values to create the Byron genesis file.
+-- The values here don't matter as the testnet conditions are ultimately determined
+-- by the Shelley genesis.
+defaultByronProtocolParamsJsonValue :: Aeson.Value
+defaultByronProtocolParamsJsonValue =
+  Aeson.object
+    [ "heavyDelThd" .= toJSON @String "300000000000"
```

Comment:
`Value` has `IsString` so I think this should work with `OverloadedStrings`:
```suggestion
    [ "heavyDelThd" .= "300000000000"
```

Reply:
Nope it doesn't :slightly_frowning_face:

```shell
src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs:567:21-22: error: [GHC-39999]
    • Ambiguous type variable ‘v0’ arising from a use of ‘.=’
      prevents the constraint ‘(ToJSON v0)’ from being solved.
      Probable fix: use a type annotation to specify what ‘v0’ should be.
      Potentially matching instances:
        instance ToJSON Aeson.Key
          -- Defined in ‘aeson-2.2.3.0:Data.Aeson.Types.ToJSON’
        instance ToJSON Aeson.DotNetTime
          -- Defined in ‘aeson-2.2.3.0:Data.Aeson.Types.ToJSON’
        ...plus 190 others
        ...plus 340 instances involving out-of-scope types
        (use -fprint-potential-instances to see them all)
    • In the expression: "heavyDelThd" .= "300000000000"
      In the first argument of ‘Aeson.object’, namely
        ‘["heavyDelThd" .= "300000000000",
          "maxBlockSize" .= toJSON @String "2000000",
          "maxTxSize" .= toJSON @String "4096",
          "maxHeaderSize" .= toJSON @String "2000000", ....]’
      In the expression:
        Aeson.object
          ["heavyDelThd" .= "300000000000",
           "maxBlockSize" .= toJSON @String "2000000",
           "maxTxSize" .= toJSON @String "4096",
           "maxHeaderSize" .= toJSON @String "2000000", ....]
    |
567 |     [ "heavyDelThd" .= "300000000000"
```

Reply:
But this does (without `OverloadedStrings` actually!):

```hs
    [ "heavyDelThd" .= ("300000000000" :: String)
```

I'm not sure this is much better than the original version, since the parentheses add noise IMO. Happy to go either way.

Reply:
Thanks for checking that. I don't have a strong opinion here, feel free to choose whichever feels more readable to you. 👍🏻

Reply:
:+1: I'm leaving the `toJSON @String` version then. Generally speaking I like more explicit code.